core/app/[locale]/layout.tsx

```diff
@@ -124,36 +123,26 @@ export default async function RootLayout({ params, children }: Props) {
     };
   });
 
+  const scripts = scriptsTransformer(rootData.data.site.content.scripts);
+
   return (
     <html className={clsx(fonts.map((f) => f.variable))} lang={locale}>
-      <head>
-        <ScriptManagerScripts
-          scripts={rootData.data.site.content.headerScripts}
-          strategy="afterInteractive"
-        />
-      </head>
       <body className="flex min-h-screen flex-col">
         <NextIntlClientProvider>
-          <ConsentManagerProvider>
+          <ConsentManager scripts={scripts}>
             <NuqsAdapter>
               <StreamableAnalyticsProvider data={streamableAnalyticsData} />
```

Comment:
🍹 Might need to consider where this analytics provider needs to live in this component tree. I would just test around to see.

Reply:
Definitely nested within `ConsentManager` since that will eventually load the gtag script required for the GA4 provider.